lib/log.py

```diff
@@ -663,6 +664,19 @@ def emit(self, record):
 In the following part of the code, logging handlers are defined
 """
 
+class EnglishLocale(logging.Formatter):
+    """
+    Use English month names for logging
+    """
+    def formatTime(self, record, datefmt=None):
```

Comment:
this function is not thread-safe. Maybe have a look at babel for a thread-safe implementation?
e.g. ``babel.dates.format_datetime(..., locale='en_US')``

Reply:
You are right. Introducing babel actually makes things quite complicated. I hope this works as expected now. Could you test please?